README.md

```diff
@@ -467,17 +471,8 @@ NNCF is also available via [conda](https://anaconda.org/conda-forge/nncf):
 conda install -c conda-forge nncf
 ```
 
-### System requirements
-
-- Ubuntu\* 18.04 or later (64-bit)
-- Python\* 3.9 or later
-- Supported frameworks:
-  - PyTorch\* >=2.4, <2.6
-  - TensorFlow\* >=2.8.4, <=2.15.1
-  - ONNX\* ==1.17.0
-  - OpenVINO\* >=2022.3.0
-
-This repository is tested on Python* 3.10.14, PyTorch* 2.5.0 (NVidia CUDA\* Toolkit 12.4) and TensorFlow* 2.12.1 (NVidia CUDA\* Toolkit 11.8).
+System requirements of NNCF corresponds with used backend, matrix of corresponding versions of backend in
```

Comment:
=> System requirements of NNCF correspond to the used backend. System requirements for each backend and the matrix of corresponding versions can be found in [installation.md](./docs/Installation.md)

Reply:
let's refer to ./docs/Installation.md not ./docs/Installation.md#corresponding-versions to cover the system requirements